src/ert/gui/tools/plot/plot_api.py

```diff
@@ -205,29 +205,37 @@ def observations_for_key(self, ensemble_ids: List[str], key: str) -> pd.DataFram
                     timeout=self._timeout,
                 )
                 self._check_response(response)
+
+                observations = response.json()
+                observations_dfs = []
+
                 if not response.json():
                     continue
                 try:
-                    obs = response.json()[0]
+                    response.json()[0]
                 except (KeyError, IndexError, JSONDecodeError) as e:
                     raise httpx.RequestError(
                         f"Observation schema might have changed key={key},  ensemble_name={ensemble.name}, e={e}"
                     ) from e
```

Comment:
It is unclear to me what the purpose of this section is now.

Reply:
I'm not sure either, but am also not confident to just remove it here as part of this bugfix, although I think this entire module, PlotAPI/Dark storage could be simplified alot

Reply:
I agree with keeping it for now